pydantic/networks.py

```diff
@@ -164,6 +164,13 @@ def import_email_validator() -> None:
         import email_validator
     except ImportError as e:
         raise ImportError('email-validator is not installed, run `pip install pydantic[email]`') from e
+    if hasattr(email_validator, '__version__') and email_validator.__version__.partition('.')[0] == '2':
+        return
+    else:
+        input_email = 'testaddr@example.tld'
+        valid_email = validate_email(input_email)
+        if not hasattr(valid_email, 'normalized'):
+            raise ImportError('email-validator version >= 2.0 required')
```

Comment:
Is there a cleaner way in which we could check for the case where the version is between `2.0.0` and `2.0.0post1`?

Reply:
normalized is the documented discriminator for 2.0
https://github.com/JoshData/python-email-validator/blob/main/CHANGELOG.md#200-april-15-2023
2.1 has __version__
https://github.com/JoshData/python-email-validator/blob/main/CHANGELOG.md#210-october-22-2023